include/exadg/structure/preconditioners/multigrid_preconditioner.cpp

```diff
@@ -110,7 +110,7 @@ MultigridPreconditioner<dim, Number>::update()
     // interpolate velocity from fine to coarse level
     this->transfer_from_fine_to_coarse_levels(
       [&](unsigned int const fine_level, unsigned int const coarse_level) {
-        auto vector_fine_level =
+        auto & vector_fine_level =
```

Comment:
Shouldn't we be able to write `auto const &` here?

Reply:
yes :)

Reply:
see #581